ecosystem/sep-0024.md

```diff
@@ -331,6 +333,13 @@ target.postMessage({
 
 As an alternative to using the `callback` parameter, the wallet can poll the transaction endpoint [`/transaction`](#single-historical-transaction) with the request `id` to check the status of the request.
 
+**Differences between `callback` and `on_change_callback`**
+
+Anchors should only send responses to `callback` when the user is finished with the interactive flow, and it's safe for the wallet to close the window. It should only be called zero or one time.
+
+`on_change_callback` should be called each time the `status` or `kyc_verified` properties of the transaction should change. It can be called any number of times.
```

Comment:
```suggestion
`on_change_callback` should be called each time the `status` or `kyc_verified` properties of the transaction change. It can be called any number of times.
```